tests/schemas/test_note_schema.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from schemas.notes import *
+from models.notes import NotesModel
+
+
+@pytest.mark.usefixtures("empty_test_db")
+class TestNotesSchemaSerialization:
+    def test_notes_serialization(self, create_join_staff, create_note):
+        user = create_join_staff()
+        note = create_note(user)
+
+        note_schema = NotesSchema()
+        note_json = note_schema.dump(note)
+        assert note_json['text'] == note.text
+        assert note_json['user'] == "%s %s" % (user.firstName, user.lastName)
+        assert not 'userinfo' in note_json
+
+    def test_notes_deserialization(self, create_join_staff):
+        note_schema = NotesSchema()
+        user = create_join_staff()
+        payload = {'user': user.id,
+                   'text': 'Test serialization',
+                   'ticketid': 1}
+        note = note_schema.load(payload)
+        assert payload['user'] == note['user']
+        assert payload['text'] == note['text']
+        assert payload['ticketid'] == note['ticketid']
+
+    def test_notes_user_validation(self):
+        note_schema = NotesSchema(exclude=["userinfo"])
+        payload = { 'user': 500,
+                    'text': "This should fail",
```

Comment:
This text comment is a bit confusing. When I first read it I thought there I would see an assertion on the text field, however it ended up checking for a valid user. Also the text field is not required for a valid_payload. I think it would be reasonable to add `nullable=False` to the text field in the model.

Reply:
Changed the text to indicate the payload is simply to validate the user.